frontend/package.json

```diff
@@ -10,7 +10,7 @@
     "buildoptimized": "NODE_ENV=production webpack-cli --config webpack.optimized.config.js",
     "build-dev": "NODE_ENV=development webpack-cli",
     "buildoptimized-dev": "NODE_ENV=development webpack-cli --config webpack.optimized.config.js",
-    "build-translation": "node i18next.scanner.js",
+    "build-translation": "node ../i18next.scanner.js",
```

Comment:
Why? isn't this file at the root of the `frontend/` folder, like `package.json`?

Reply:
I moved it to the root folder since:
- the scanner options were already at root
- this avoids forgetting one app if we need to update this file (previously some `i18n.scanner.js` were taking all .js files into account, but not all).